client/network/src/peerset.rs

```diff
@@ -338,14 +312,6 @@ impl Peerset {
 		self.protocol_handles[set_id.0].dropped(peer_id);
 	}
 
-	/// Reports an adjustment to the reputation of the given peer.
-	pub fn report_peer(&mut self, peer_id: PeerId, score_diff: ReputationChange) {
```

Comment:
Why is this removed?

Reply:
All peer reporting is happening through the `PeersetHandle` now, so this function is not used. I don't know why the compiler didn't complain before.